src/main/resources/db/migration/V1_83__expense_report_details.sql

```diff
@@ -0,0 +1,190 @@
+                         aa.appearance_stage,
+                         aa.attendance_type,
+                         aa.is_draft_expense,
+                         aa.pay_cash,
+                         aa.f_audit,
+                         aa.attendance_audit_number,
+
+                         COALESCE(aa.public_transport_total_due, 0::numeric)  as public_transport_total_due,
+                         COALESCE(aa.hired_vehicle_total_due, 0::numeric)     as hired_vehicle_total_due,
+                         COALESCE(aa.motorcycle_total_due, 0::numeric)        as motorcycle_total_due,
+                         COALESCE(aa.car_total_due, 0::numeric)               as car_total_due,
+                         COALESCE(aa.pedal_cycle_total_due, 0::numeric)       as pedal_cycle_total_due,
+                         COALESCE(aa.parking_total_due, 0::numeric)           as parking_total_due,
+                         COALESCE(aa.loss_of_earnings_due, 0::numeric)        as loss_of_earnings_due,
+                         COALESCE(aa.childcare_total_due, 0::numeric)         as childcare_total_due,
+                         COALESCE(aa.misc_total_due, 0::numeric)              as misc_total_due,
+                         COALESCE(aa.subsistence_due, 0::numeric)             as subsistence_due,
+                         COALESCE(aa.smart_card_due, 0::numeric)              as smart_card_due,
+
+                         COALESCE(aa.public_transport_total_paid, 0::numeric) as public_transport_total_paid,
+                         COALESCE(aa.hired_vehicle_total_paid, 0::numeric)    as hired_vehicle_total_paid,
+                         COALESCE(aa.motorcycle_total_paid, 0::numeric)       as motorcycle_total_paid,
+                         COALESCE(aa.car_total_paid, 0::numeric)              as car_total_paid,
+                         COALESCE(aa.pedal_cycle_total_paid, 0::numeric)      as pedal_cycle_total_paid,
+                         COALESCE(aa.parking_total_paid, 0::numeric)          as parking_total_paid,
+                         COALESCE(aa.loss_of_earnings_paid, 0::numeric)       as loss_of_earnings_paid,
+                         COALESCE(aa.childcare_total_paid, 0::numeric)        as childcare_total_paid,
+                         COALESCE(aa.misc_total_paid, 0::numeric)             as misc_total_paid,
+                         COALESCE(aa.subsistence_paid, 0::numeric)            as subsistence_paid,
+                         COALESCE(aa.smart_card_paid, 0::numeric)             as smart_card_paid
+
+                  from juror_mod.financial_audit_details fad
+                           join juror_mod.financial_audit_details_appearances fada
+                                on fada.financial_audit_id = fad.id and fada.loc_code = fad.loc_code
+                           join juror_mod.appearance_audit aa
+                                on aa.loc_code = fada.loc_code and aa.attendance_date = fada.attendance_date and
```

Comment:
see - we're using the existing loc_code here to join onto financial audit details 😛